### ISSUE TITLE
Q. TraceViewer used to show aggregate information about event parameters

### ISSUE THREAD

**Issue by uucidl** [catapult-project/catapult#3101]
Trace Viewer in Chrome used to show aggregate information about numeric parameters, which I don't see anymore in the latest update of chrome.

Instead I see an histogram for durations etc... but no summary of values anymore.

This is for event 'I' types, on which some values are collected.

**Comment by eakuefner** [comment]
@uucidl Can you upload some screenshots for context?

@benshayden

**Comment by benshayden** [comment]
@lpy recently changed the [multi-event-sub-view](https://github.com/catapult-project/catapult/blob/master/tracing/tracing/ui/analysis/multi_event_sub_view.html) to display a histogram instead of a table for performance reasons. Is that what you're talking about, @uucidl ?

You should be able to click and drag across the bars in the histogram in order to display the table of values for the selected events.

I agree that drag gesture that may not be very discoverable. Maybe a tooltip would help? Like "Click and drag to select histogram bars in order to see more information about selected samples."?

**Comment by uucidl** [comment]
Indeed! I followed your instructions and clicked on the bars when showing the histogram for the Start of my events, and after doing so I could scroll down to reveal the list of all events with their arguments, which is what I was after.

What makes it hard to discover is that bars would normally not behave like a reveal button, and even after clicking there's no indication that content has been revealed at the bottom (especially with modern style scrollbars which only flash or activate during gestures)

**Comment by uucidl** [comment]
So I would encourage any effort in making this more discoverable!
